### PR TITLE
add: entity annotations, repo components

### PR DESCRIPTION
**Pull request by qpco**
also added scan annotations in PotlukkApp.java

### CODE REVIEW

File: src/main/java/dev/group1/Potlukk/repos/ItemRepo.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package dev.group1.Potlukk.repos;
+
+import dev.group1.Potlukk.entities.Item;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+
+@Component
+@Repository
+public interface ItemRepo extends JpaRepository<Item, Integer> {
```
Comment (GraysonHoward) [inline src/main/java/dev/group1/Potlukk/repos/ItemRepo.java:10]: I think we'll want a way to get all the items from a specific potlukk.

Comment (GraysonHoward) [reply src/main/java/dev/group1/Potlukk/repos/ItemRepo.java:13]: Spring will do this for you if you look at adam's pleasentSpring repo you can see an example of what I'm talking about

Comment (qpco) [reply src/main/java/dev/group1/Potlukk/repos/ItemRepo.java:13]: oh I see what you mean. i'll replace it

**Review by ZanettaR — APPROVED** [review]

**Review by GraysonHoward — APPROVED** [review]
I think everything here looks good now

**Pull request merged**